### ISSUE TITLE
Pages being duplicated in list when using accents

### ISSUE THREAD

**Issue by lsgpimentel** [jaap-karssenberg/zim-desktop-wiki#171]
Hi,

When I create a page with an accent in any letter (e.g. `á`, `é`, `ã`), the page shows duplicated in the list.

OS: OS X 10.12.6
ZIm 0.67

**Comment by jaap-karssenberg** [comment]
Can't reproduce it, either OS X specific or I need more detailed instructions how to show the issue

**Comment by lsgpimentel** [comment]
@jaap-karssenberg 

![image](https://user-images.githubusercontent.com/1979926/30725361-a44fc1e8-9f1a-11e7-8664-5c3f617c8e0a.png)


Debug messages (creating and deleting the page):
```

DEBUG: BackgroundCheck started
DEBUG: BackgroundCheck finished
DEBUG: BackgroundCheck started
DEBUG: BackgroundCheck finished
DEBUG: Action: new_sub_page
DEBUG: Opening dialog "New Sub Page - Zim"
DEBUG: Dialog response OK
INFO: Loading template from: /Users/XXXXX/.local/share/zim/templates/wiki/Default.txt
INFO: Open page: Home:É (Home:É)
DEBUG: Saving page: Home:É
DEBUG: Store page: Home:É
DEBUG: Index file: /Users/XXXXX/Documents/Zim/Notebooks/TestNotebook/Home/É.txt
DEBUG: Closed dialog "New Sub Page"
DEBUG: BackgroundCheck started
DEBUG: BackgroundCheck found out-of-date
DEBUG: Index folder: /Users/XXXXX/Documents/Zim/Notebooks/TestNotebook
DEBUG: Index folder: /Users/XXXXX/Documents/Zim/Notebooks/TestNotebook/Home
DEBUG: Index file: /Users/XXXXX/Documents/Zim/Notebooks/TestNotebook/Home/É.txt
DEBUG: Index file: /Users/XXXXX/Documents/Zim/Notebooks/TestNotebook/Home/É.txt
DEBUG: BackgroundCheck started
DEBUG: BackgroundCheck finished
DEBUG: Action: delete_page
DEBUG: Opening ProgressDialog: Removing Links
DEBUG: Trash page: Home:É
INFO: Move /Users/XXXXX/Documents/Zim/Notebooks/TestNotebook/Home/É.txt to trash
DEBUG: Drop file: Home/É.txt
INFO: Open page: Home (Home)
DEBUG: Found template 'Default' for Home
INFO: Loading template from: /Users/XXXXX/.local/share/zim/templates/wiki/Default.txt
DEBUG: BackgroundCheck started
DEBUG: BackgroundCheck found out-of-date
DEBUG: Index folder: /Users/XXXXX/Documents/Zim/Notebooks/TestNotebook
DEBUG: Drop file: Home/É.txt
DEBUG: Drop folder: Home
DEBUG: BackgroundCheck started
DEBUG: BackgroundCheck finished
```

**Comment by jaap-karssenberg** [comment]
Believe this to be fixed in current master branch / to be released 0.70 version
